Client/Clique/Modules/CreatedEvents/CreatedEventsViewController.swift

```diff
@@ -8,7 +8,72 @@
 import UIKit
 
 class CreatedEventsViewController: UIViewController {
+    
+    @IBOutlet private var _tableView : UITableView?
+    
+    private var _events : [Event] = [] {
+        didSet {
+            _tableView?.reloadData()
+        }
+    }
+    private var _eventServices = EventServices()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        setupUI()
+        _getEvents()
+    }
+}
+
+private extension CreatedEventsViewController {
+    func setupUI(){
+        _tableView?.delegate = self
+        _tableView?.dataSource = self
```

Comment:
This is what I've been referring to - use this approach or one in the storyboard.